default.json

```diff
@@ -36,36 +36,16 @@
       "groupName": "kustomize"
     },
     {
-      "matchManagers": ["dockerfile", "helm-values"],
-      "matchPackagePatterns": ["registry.suse.com/"],
-      "matchUpdateTypes": ["patch", "pin", "digest"],
-      "schedule": ["every weekend after 4am"]
+      "matchPackagePatterns": ["registry.suse.com/bci/bci"],
+      "allowedVersions": "<15.5.0"
     },
     {
       "matchPackagePatterns": ["rancher/dapper"],
-      "allowedVersions": "<1.0.0",
-      "schedule": ["every weekend after 4am"]
-    },
-    {
-      "matchPackagePatterns": ["golangci/golangci-lint"],
-      "matchUpdateTypes": ["minor", "patch", "pin", "digest"],
-      "schedule": ["every weekend after 4am"]
-    },
-    {
-      "matchManagers": ["droneci"],
-      "matchPackagePatterns": ["plugins/"],
-      "matchUpdateTypes": ["minor", "patch", "pin", "digest"],
-      "schedule": ["every weekend after 4am"]
+      "allowedVersions": "<1.0.0"
     },
     {
       "matchManagers": ["github-actions"],
-      "matchUpdateTypes": ["minor", "patch"],
-      "pinDigests": true,
-      "schedule": ["every weekend after 4am"]
-    },
-    {
-      "matchPackagePatterns": ["derailed/k9s"],
```

Comment:
Not sure the title/commit aligns with the changes regarding `derailed/k9s`, `droneci`, and `golangci/golangci-lint`. Please add some context on the PR description.